src/app/learn/get-started/prereqs/initial-setup/index.html

```diff
@@ -12,27 +13,27 @@
   <stache-page-anchor>
     Node.js
   </stache-page-anchor>
-
-  <p>
-    Install the latest LTS release of <a href="https://nodejs.org/en/download/releases/">Node.js</a>. We require at least version 6. To check your version, run <stache-code>node –v</stache-code> from the command line.
-  </p>
   <p>
-    For Mac OS X and Linux, we recommend that you use <a href="https://github.com/creationix/nvm">Node Version Manager (nvm)</a> to wrap your NodeJS installation so that it installs in your user directory and avoids permissions-related issues.
+    SKY UX requires an LTS Dubnium version of Node.js, and we recommend <a href="https://nodejs.org/en/download/releases/">the latest release</a>. To check your version, run <stache-code>node –v</stache-code> from the command line.
   </p>
   <p>
-    For Windows, download the <stache-code>msi</stache-code> version of the latest LTS release. Windows machines also require two additional steps:
+    In addition, SKY UX recommends Node Version Manager (NVM) to wrap your Node.js installation so that it installs in your user directory and avoids permissions-related issues.
```

Comment:
There's an extra benefit I would like to highlight.  Perhaps:
```
In addition, SKY UX recommends Node Version Manager (NVM) to wrap your Node.js installation so that it installs in your user directory and avoids permissions-related issues as well as allowing you to easily upgrade and manage multiple Node.js versions.
```

Or however you could word that better.

Reply:
Sounds good. Updating now.